### ISSUE TITLE
[FEAT] Adding support for strikethrough/custom cloze

### ISSUE THREAD

**Issue by dummifiedme** [st3v3nmw/obsidian-spaced-repetition#387]
**Is your feature request related to a problem? Please describe.**
Currently highlights are utilized which turn out to ne impotant as visual cues for me while reading. If we could have custom clozes, we could have out own symbols to work with!!

**Describe the solution you'd like*
We could have any set of symbols for this plug-in to search for. We can just enclose the cloze in those symbols which we want to assign.
eg `__cloze__`or `~~ cloze~~`or `₹₹cloze₹₹`Anything really.

**Comment by AB1908** [comment]
I believe cloze symbols can be altered in settings.

**Comment by dummifiedme** [comment]
> I believe cloze symbols can be altered in settings.

there are two modes, bold or highlight. Thats it.

**Comment by AB1908** [comment]
Ah, you're right. I confused this with other settings.

**Comment by AB1908** [comment]
I think I might be able to add this but will need time.

**Comment by dummifiedme** [comment]
> 



> I think I might be able to add this but will need time.

If you cn please do. Happy to wait..

**Comment by st3v3nmw** [comment]
Hi,

I've just made a release [v1.13.0](https://github.com/st3v3nmw/obsidian-spaced-repetition/releases/tag/1.13.0) that ships with custom cloze patterns 🥳 all thanks to #943. The updated documentation is live [here](https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/cloze-cards/#cloze-types).

I've tested the release but not as extensively as I normally would. Perhaps we need a beta version of the plugin? That being said, I'm on standby to fix bugs ASAP. Just give me a shout.

Cheers.
